src/HDInsight/HDInsight/Models/Management/AzureHdInsightManagementClient.cs

```diff
@@ -121,16 +122,33 @@ public virtual OperationResource DeleteCluster(string resourceGroupName, string
             return HdInsightManagementClient.Clusters.Delete(resourceGroupName, clusterName);
         }
 
+        [Obsolete("This method has been deprecated and will stop working.")]
         public virtual OperationResource ConfigureHttp(string resourceGroupName, string clusterName, HttpSettingsParameters httpSettings)
         {
             return HdInsightManagementClient.Clusters.ConfigureHttpSettings(resourceGroupName, clusterName, httpSettings);
         }
 
+        public virtual OperationResource UpdateGatewayCredential(string resourceGroupName, string clusterName, HttpSettingsParameters httpSettings)
+        {
+            return HdInsightManagementClient.Clusters.UpdateGatewaySettings(resourceGroupName, clusterName, httpSettings);
+        }
+
+        [Obsolete("This method has been deprecated and will stop working.")]
```

Comment:
same

Reply:
> same

Fixed.

Reply:
fixed

---
In reply to: [278675897](https://github.com/Azure/azure-powershell/pull/9069#discussion_r278675897) [](ancestors = 278675897)